Marlin/src/lcd/language/language_en.h

```diff
@@ -427,8 +427,10 @@ namespace LanguageNarrow_en {
   LSTR MSG_SHAPING_ZETA                   = _UxGT("@ damping");
   LSTR MSG_SHAPING_A_FREQ                 = STR_A _UxGT(" frequency");  // ProUI
   LSTR MSG_SHAPING_B_FREQ                 = STR_B _UxGT(" frequency");  // ProUI
+  LSTR MSG_SHAPING_C_FREQ                 = STR_B _UxGT(" frequency");  // ProUI
   LSTR MSG_SHAPING_A_ZETA                 = STR_A _UxGT(" damping");    // ProUI
   LSTR MSG_SHAPING_B_ZETA                 = STR_B _UxGT(" damping");    // ProUI
+  LSTR MSG_SHAPING_C_ZETA                 = STR_B _UxGT(" damping");    // ProUI
```

Comment:
I think these should be `STR_C`?

Reply:
I know you saw it, but just closing the loop on this: https://github.com/MarlinFirmware/Marlin/pull/27119

Reply:
oh no, now i feel like a dummy. totally overlooked that